docs/analyzer.md

```diff
@@ -0,0 +1,25 @@
+# Analyzer
+
+FakeItEasy provides a Roslyn analyzer to detect incorrect usages of the library,
+which cannot be prevented by the API or the compiler but will typically result in
+bugs at runtime.

```

Comment:
Ah, yes,"that" sounds better. After 25 years of learning and speaking English, I *still* don't know when I should use "that" or "which"...